### ISSUE TITLE
outfit() rules do not work properly.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [arkayenro/arkinventory#290]
```
Downloaded from > googlecode | wowinterface | curse | wowui | other (specify)


What steps will reproduce the problem?
1. Have multiple equipment sets.
2. Make a rule for each set.
3. make a rule for epic equipment, and for blue equipment.
4. assign a bar to each of those rules.
5. Take an item from the non-outfit bar, equip it, save the outfit.
6. Notice that the swapped item do not show up in the correct bar if
   the swapped item is in another set.
   For instance, I now have an item that matches rule "outfit(Heals)"
   sitting in the general epic bar and not in the Heals bar.

What is the expected output? What do you see instead?

If items are in outfits then when they are swapped or have their
outfit locations changed they should appear in the proper bar.

basically you have bars with rules that show items that match
a different, higher order (lower number) rule.

My guess is that you are missing a trigger to recalculate
bar assignments of items.

What version of the product are you using? On what operating system?

3.2.24

Please provide any additional information below.
ugh.

```

Original issue reported on code.google.com by `druber...@gmail.com` on 6 Nov 2009 at 7:05
